src/Context/ContentContext.php

```diff
@@ -290,6 +290,12 @@ protected function createContent($contentType, array $data): void
                 case '_hidden':
                     $locationStruct->hidden = (bool)$value;
                     break;
+                case '_sortField':
+                    $locationStruct->sortField = (int)$value;
+                    break;
+                case '_sortOrder':
+                    $locationStruct->sortOrder = (int)$value;
```

Comment:
Would be better to automatically convert "ASC" and "DESC" to it's according int values. That would make the feature much more readable.

Reply:
Yes, you're right! I changed it so the features now use readable strings according to the constants defined in the Location class.